automathemely/autoth_tools/updsuntimes.py

```diff
@@ -5,7 +5,7 @@
 
 import pytz
 import tzlocal
-from astral import Location
+from astral import location
```

Comment:
I think this should be instead
```suggestion
from astral.location import Location
```
Otherwise, the code on line 66 of the same file will fail.